lib/library.rb

```diff
@@ -0,0 +1,57 @@
+require 'date'
+require 'yaml'
+
+class Library
+  attr_accessor :items
+
+  def initialize(options={})
```

Comment:
Lint/UnusedMethodArgument: Unused method argument - `options`. If it's necessary, use `_` or `_options` as an argument name to indicate that it won't be used. You can also write as `initialize(*)` if you want the method to accept any arguments but don't care about them. (https://github.com/bbatsov/ruby-style-guide#underscore-unused-vars)